storage/src/mmr/benches/append.rs

```diff
@@ -16,9 +16,10 @@ fn bench_append(c: &mut Criterion) {
         // Append elements to MMR
         c.bench_function(&format!("{}/n={}", module_path!(), n), |b| {
             b.iter(|| {
+                let mut h = Sha256::new();
                 let mut mmr = Mmr::<Sha256>::new();
                 for digest in &elements {
-                    mmr.add(digest);
+                    mmr.add_with_hasher(digest, &mut h);
```

Comment:
If we keep both `add` and `add_with_hasher`, I think we should consider benchmarking both.

Reply:
changed to just have one